tools/make/lib/lint/typescript/Makefile

```diff
@@ -32,19 +32,18 @@ include $(TOOLS_MAKE_LIB_DIR)/lint/typescript/dtslint.mk
 # -   This rule supports the environment variables supported by each context-specific (`lint-typescript-<context>`) prerequisite.
 # -   This rule is useful when wanting to glob for files, irrespective of context, for a particular package in order to lint all contained TypeScript files.
 #
-# @param {string} [TYPESCRIPT_SOURCES_FILTER] - file path pattern (e.g., `.*/blas/base/dasum/.*`)
-# @param {string} [TYPESCRIPT_TESTS_FILTER] - file path pattern (e.g., `.*/blas/base/dasum/.*`)
+# @param {string} [SOURCES_FILTER] - file path pattern (e.g., `.*/blas/base/dasum/.*`)
 # @param {string} [TYPESCRIPT_DECLARATIONS_FILTER] - file path pattern (e.g., `.*/blas/base/dasum/.*`)
-# @param {string} [TYPESCRIPT_EXAMPLES_FILTER] - file path pattern (e.g., `.*/blas/base/dasum/.*`)
+# @param {string} [TYPESCRIPT_DECLARATIONS_TESTS_FILTER] - file path pattern (e.g., `.*/blas/base/dasum/.*`)
 # @param {*} [FAST_FAIL] - flag indicating whether to stop linting upon encountering a lint error
 #
 # @example
 # make lint-typescript
 #
 # @example
-# make lint-typescript TYPESCRIPT_SOURCES_FILTER=".*/blas/base/dasum/.*" TYPESCRIPT_TESTS_FILTER=".*/blas/base/dasum/.*" TYPESCRIPT_DECLARATIONS_FILTER=".*/blas/base/dasum/.*" TYPESCRIPT_EXAMPLES_FILTER=".*/blas/base/dasum/.*"
+# make lint-typescript SOURCES_FILTER=".*/blas/base/dasum/.*" TYPESCRIPT_DECLARATIONS_FILTER=".*/blas/base/dasum/.*" TYPESCRIPT_DECLARATIONS_TESTS_FILTER=".*/blas/base/dasum/.*"
 #/
-lint-typescript: lint-typescript-src lint-typescript-tests lint-typescript-examples lint-typescript-declarations
+lint-typescript: lint-typescript-src lint-typescript-declarations lint-typescript-declarations-tests
```

Comment:
Okay. After getting myself thoroughly confused and now reacquainting myself with the environment variable [Makefile](https://github.com/stdlib-js/stdlib/blob/ae9cdb7f4a5bad86dfb5f67f3ea7656e362a71a7/tools/make/lib/ls/Makefile), I understand better the changes made in this file.

FWIW, IIRC, I had intentionally added `src`, `tests`, and `examples` rules for "future proofing". The only rules that we actually care about here are `declarations` and `declarations-tests`. We're not likely to have any "source" TypeScript files, either.

So if you wanted to remove `lint-typescript-src`, that would be fine, as well. We can always re-add these rules later.

Reply:
Yeah, I thought it was confusing and we can always add them back should we need them. I personally would be in favor of also removing the `-src` make directive here.

Reply:
Go for it.

Reply:
Done.